### DISCUSSION TITLE
PAG从4.2.55版本开始有联网操作 #2511

### DISCUSSION THREAD

**Discussion by hnubility** [Tencent/libpag#2512] [Q&A] [answered]
低版本创建decoder不会释放，导致系统卡顿

更新到最新后能够解决问题，但是为什么会有网络请求呢

**Comment by kevingpqi123** [comment, marked as answer]
可以看下下面的版本更新说明和 PAG SDK个人信息保护规则
https://github.com/Tencent/libpag/releases/tag/v4.0.5.51

https://privacy.qq.com/document/preview/01e79d0cc7a2427ba774b88c6beff0fd

**Reply by hnubility** [reply]
想问下有没有低版本有办法可以正确释放Media Decoder;使用网络要求需要告知用户，不想用有联网的版本号

**Reply by kevingpqi123** [reply]
pag 的源码是开源的，那你需要自己编译构建版本，我们需要一些业务接入数量向老板申请资源，如果没有资源，pag 的人力维护可能要进一步缩减，包含现在有限的技术支持
